backend/endpoints/posts.js

```diff
@@ -88,6 +91,27 @@ module.exports = function (app) {
     res.status(200).json();
   });
 
+  /***
+   * Gets a post based on postID.
+   */
+  app.get("/post/:postID", async (req, res) => {
+    const postID = req.params.postID;
+
+    let postData = await prisma.post.findUnique({
+      where: { id: Number(postID) },
+    });
+
+    const comments = await prisma.comment.findMany({
+      where: {
+        postID: Number(postID),
+      },
+    });
+
+    postData = { ...postData, comments: comments };
```

Comment:
Using the same variable name could cause confusion, as these two usages imply different contents